client/ayon_core/version.py

```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 """Package declaring AYON core addon version."""
-__version__ = "0.3.1-dev.2"
+__version__ = "0.3.2-dev.2"
```

Comment:
it seems that this is not the commit for this PR.

Reply:
This PR updates default values of `ayon+settings://core/tools/creator/product_name_profiles`

Reply:
```suggestion
__version__ = "0.3.1-dev.2"
```

Reply:
https://github.com/ynput/ayon-core/pull/456/commits/888c4ff35434588fc28cc468753a1ac9ce05001f